### ISSUE TITLE
bugs when train

### ISSUE THREAD

**Issue by cuge1995** [PeizeSun/SparseR-CNN#6]
## Instructions To Reproduce the 🐛 Bug:
1. Full runnable code or full changes you made:
```shell
python projects/SparseR-CNN/train_net.py --num-gpus 8 \
    --config-file projects/SparseR-CNN/configs/sparsercnn.res50.100pro.3x.yaml
```
3. __Full logs__ you observed:
```
Traceback (most recent call last):
  File "projects/SparseR-CNN/train_net.py", line 22, in <module>
    from detectron2.data import MetadataCatalog, build_detection_train_loader
  File "/home/amax/ngrok/opencv/SparseR-CNN/detectron2/data/__init__.py", line 4, in <module>
    from .build import (
  File "/home/amax/ngrok/opencv/SparseR-CNN/detectron2/data/build.py", line 12, in <module>
    from detectron2.structures import BoxMode
  File "/home/amax/ngrok/opencv/SparseR-CNN/detectron2/structures/__init__.py", line 7, in <module>
    from .masks import BitMasks, PolygonMasks, rasterize_polygons_within_box, polygons_to_bitmask
  File "/home/amax/ngrok/opencv/SparseR-CNN/detectron2/structures/masks.py", line 9, in <module>
    from detectron2.layers.roi_align import ROIAlign
  File "/home/amax/ngrok/opencv/SparseR-CNN/detectron2/layers/__init__.py", line 3, in <module>
    from .deform_conv import DeformConv, ModulatedDeformConv
  File "/home/amax/ngrok/opencv/SparseR-CNN/detectron2/layers/deform_conv.py", line 11, in <module>
    from detectron2 import _C
ImportError: /home/amax/ngrok/opencv/SparseR-CNN/detectron2/_C.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZN6caffe28TypeMeta21_typeMetaDataInstanceISt7complexIdEEEPKNS_6detail12TypeMetaDataEv
```

## Environment:

Provide your environment information using the following command:
```
----------------------  ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
sys.platform            linux
Python                  3.6.10 |Anaconda, Inc.| (default, May  8 2020, 02:54:21) [GCC 7.3.0]
numpy                   1.19.1
detectron2              0.3 @/home/amax/ngrok/opencv/SparseR-CNN/detectron2
detectron2._C           not built correctly: /home/amax/ngrok/opencv/SparseR-CNN/detectron2/_C.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZN6caffe28TypeMeta21_typeMetaDataInstanceISt7complexIdEEEPKNS_6detail12TypeMetaDataEv
Compiler                c++ (Ubuntu 6.5.0-2ubuntu1~16.04) 6.5.0 20181026
CUDA compiler           Cuda compilation tools, release 10.2, V10.2.89
DETECTRON2_ENV_MODULE   <not set>
PyTorch                 1.7.0 @/home/amax/anaconda3/envs/pcdet/lib/python3.6/site-packages/torch
PyTorch debug build     True
GPU available           True
GPU 0,1                 GeForce RTX 2080 Ti (arch=7.5)
CUDA_HOME               /usr/local/cuda-10.2
Pillow                  7.2.0
torchvision             0.8.1 @/home/amax/anaconda3/envs/pcdet/lib/python3.6/site-packages/torchvision
torchvision arch flags  3.5, 5.0, 6.0, 7.0, 7.5
fvcore                  0.1.2.post20201122
cv2                     Not found
----------------------  ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2020.0.0 Product Build 20191122 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v1.6.0 (Git Hash 5ef631a030a6f73131c77892041042805a06064f)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 10.2
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75
  - CuDNN 7.6.5
  - Magma 2.5.2
  - Build settings: BLAS=MKL, BUILD_TYPE=Release, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DUSE_VULKAN_WRAPPER -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, USE_CUDA=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON,
```

**Comment by PeizeSun** [comment]
I guess this issue may help you :
https://github.com/facebookresearch/maskrcnn-benchmark/issues/891

**Comment by cuge1995** [comment]
solved by using `torch==1.5 torchvision==0.6`
